app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,480 @@ def navigation
         end
         navigation_art
       end
-      
-      
+
       def content_common
         {
-          :name => t('site.channels.' + @channel.id.to_s + '.title'),
-          :description => t('site.channels.' + @channel.id.to_s + '.description')
+          name: t('site.channels.' + @channel.id.to_s + '.title'),
+          description: t('site.channels.' + @channel.id.to_s + '.description')
         }
       end
-      
 
-      
-      def content_art        
-        {          
-          :channel_info  => {
-            :name => content_common[:name],
-            :description => content_common[:description],
-            :stats  => {
-              :items => [
+      def content_art
```

Comment:
Assignment Branch Condition size for content_art is too high. [35/15]<br>Method has too many lines. [184/10]